.ci/docker/common/install_graphviz.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+source "$(dirname "${BASH_SOURCE[0]}")/common_utils.sh"
+
+if [ -n "${UBUNTU_VERSION}" ]; then
+    apt update
+    apt-get install -y graphviz
+fi
```

Comment:
Would be nice to raise an error here, if someone tries to call from CentOS/RHEL system